sysdump/sysdump.go

```diff
@@ -2116,6 +2133,44 @@ func (c *Collector) SubmitProfilingGopsSubtasks(pods []*corev1.Pod, containerNam
 	return nil
 }
 
+// SubmitTracingGopsSubtask submits task to collect tracing data from pods.
+func (c *Collector) SubmitTracingGopsSubtask(pods []*corev1.Pod, containerName string) error {
+	for _, p := range pods {
+		p := p
+		if err := c.Pool.Submit(fmt.Sprintf("gops-%s-%s", p.Name, gopsTrace), func(ctx context.Context) error {
+			agentPID, err := c.getGopsPID(ctx, p, containerName)
+			if err != nil {
+				return err
+			}
+			o, err := c.Client.ExecInPod(ctx, p.Namespace, p.Name, containerName, []string{
+				gopsCommand,
```

Comment:
I think this would be more readable as

```
gopsPath,    // replaces "Command"
gopsCommand, // replaces "Trace"
```

Reply:
Actually, we have other gops sub-commands like `gopsProfiling`. An alternative would be to just swap `gopsCommand` for `gopsPath`, but I'm not sure this is truly better, as gops is used also as a command itself to detect the PID: https://github.com/cilium/cilium-cli/blob/0245001fd6fd9c907780ef0b78547caf118c72eb/sysdump/sysdump.go#L2046-L2050

WDYT?

Reply:
Given the different named, subcommands, I think "Command" is fine. Resolving :)